pkg/formatted/k8s.go

```diff
@@ -29,11 +32,24 @@ var ConditionColor = map[string]color.Attribute{
 	"Pending":   color.FgHiWhite,
 }
 
+var stepCounter uint64
+
 // ColorStatus Get a status coloured
 func ColorStatus(status string) string {
 	return color.New(ConditionColor[status]).Sprint(status)
 }
 
+// AutoStepName when our stepName is empty return a genrated name as generated
+// on pipeLine
+func AutoStepName(stepName string) string {
+	if stepName != "" {
```

Comment:
I do this because I could not find a way to do this nicely in builtin gotemplate....

Reply:
Just making sure .. so we want to print something like .. 

```
- A named step
- unnamed-2
- another-named-step
- unnamed-4
```
- right?

Reply:
Just saw that the test confirms it and it makes sense!